src/ros_filter.cpp

```diff
@@ -706,6 +743,9 @@ namespace RobotLocalization
     nhLocal_.param("smooth_lagged_data", smoothLaggedData_, false);
     nhLocal_.param("history_length", historyLength_, 0.0);
 
+    // Wether we reset filter on jump back in time
+    nhLocal_.param("reset_on_jump_back_in_time", resetOnJumpBackInTime_, false);
```

Comment:
I also didn't had a good idea. I going to through in some ideas, what about:
1. jump_back_reset
2. time_jump_reset
3. reset_on_time_jump
4. reset_time_jump
5. reset_on_loop
6. reset_loop

I really don't know. Pick one :-)